src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -72,7 +76,7 @@ public Command testSpeed(double speed) {
 
     public CommandBase setTargetAndStartShooter(HubSpeed hub) {
         return instant("Set Default Speed", () -> {
-            shootSpeed = hub.get();
+            shootSpeed = highGoal;
```

Comment:
Why even pass in the hub parameter if you're not going to use it?

Reply:
The `highGoal` variable is only used to dynamically change it through the SmartDashboard. After we find the proper speed for the high goal, the command will use the `HubSpeed` enum.